modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
+set of CPUs. This is useful for resource-constrained environments, such as single-node production deployments,
+where you want to reserve most of the CPU resources for user workloads and configure OpenShift to run on a fixed
+number of CPUs within the host.
+
+Server resources deployed at the edge, such as cores, are expensive and limited. Customers require nearly all cores for their application workloads. The resources consumed by infrastructure is a key portion of the customer's business
```

Comment:
This para reads like we are repeating back to our customers what they've already told Red Hat.  Can it be respun so that what we're offering enables customers to maximize their hardware?

Reply:
Reworded. "Server resources deployed at the edge, such as cores, are expensive and limited. Application workloads require nearly all cores and the resources consumed by infrastructure is a key reason for the selection of a vRAN infrastructure. A hypothetical distributed unit (DU) example is an unusually resource-intensive workload,typically 
requiring 20 dedicated cores. Partitioning management workloads mitigates much of this activity by separating management tasks from normal workloads."